examples/forecasting/advanced_forecasting_example.py

```diff
@@ -0,0 +1,65 @@
+from autogluon.forecasting import ForecastingPredictor
+from autogluon.forecasting import TabularDataset
+import autogluon.core as ag
+
+train_data = TabularDataset("https://autogluon.s3-us-west-2.amazonaws.com/datasets/CovidTimeSeries/train.csv")
+test_data = TabularDataset("https://autogluon.s3-us-west-2.amazonaws.com/datasets/CovidTimeSeries/test.csv")
+static_features = TabularDataset("./COV19/toy_static_features.csv")
+# change this to specify search strategy, can try bayesopt, random, or skopt
+searcher_type = "random"
+# change this to specify eval metric, one of ["MASE", "MAPE", "sMAPE", "mean_wQuantileLoss"]
+eval_metric = "mean_wQuantileLoss"
+
+prediction_length = 19
+path = "hpo_models"
+
+predictor = ForecastingPredictor(path=path, eval_metric=eval_metric).fit(train_data,
+                                                                         prediction_length,
+                                                                         index_column="name",
+                                                                         target_column="ConfirmedCases",
+                                                                         time_column="Date",
+                                                                         hyperparameter_tune=True,
+                                                                         quantiles=[0.1, 0.5, 0.9],
+                                                                         refit_full=True,
+                                                                         keep_only_best=True,
+                                                                         set_best_to_refit_full=True,
+                                                                         search_strategy=searcher_type,
+                                                                         static_features=static_features,
```

Comment:
For clearer presentation, I would move the static_features argument next to train_data, since they are both data-related args.

Reply:
We cannot move static features immediately after train_data as prediction_length is a mandatory kwarg and static_features is an optional one so prediction_length must be ahead of static_features. I can put it after prediction_length.

Reply:
Yes sorry that's what I meant. Just move all the data-related arguments to be as close together as possible.